tom_tns/tns_api.py

```diff
@@ -282,19 +282,23 @@ def get_tns_report_reply(report_id, request):
     Posts an informational message in a banner on the page using ``request``
     """
     tns_info = get_tns_credentials()
+    max_attempts = tns_info.get('report_max_attempts', 10)
+    delay_seconds = tns_info.get('report_delay_seconds')
     reply_data = {'api_key': tns_info['api_key'], 'report_id': report_id}
     iau_name = None
     attempts = 0
     # TNS Submissions return immediately with an id, which you must then check to see if the message
     # was processed, and if it was accepted or rejected. Here we check up to 10 times, waiting 1s
```

Comment:
This comment should be updated to reflect that the delay seconds scales up linearly by default and is not just 1 second by default anymore.